test/e2e/playwright/consolidation-orders.spec.ts

```diff
@@ -76,7 +77,6 @@ test.describe('Consolidation Orders', () => {
       `#loading-spinner-case-assignment-${firstChildCaseId}`,
       timeoutOption,
     );
-    await page.waitForSelector(`#case-assignment-${firstChildCaseId}`, timeoutOption);
```

Comment:
We would need to have attorney assignments on all seeded cases in order for this to function properly 100% of the time.